connect/basic-auth-extension/src/test/java/org/apache/kafka/connect/rest/basic/auth/extension/BasicAuthSecurityRestExtensionTest.java

```diff
@@ -55,20 +58,21 @@ public void tearDown() {
     @SuppressWarnings("unchecked")
     @Test
     public void testJaasConfigurationNotOverwritten() {
-        Capture<JaasBasicAuthFilter> jaasFilter = EasyMock.newCapture();
-        Configurable<? extends Configurable<?>> configurable = EasyMock.mock(Configurable.class);
-        EasyMock.expect(configurable.register(EasyMock.capture(jaasFilter))).andReturn(null);
-  
-        ConnectRestExtensionContext context = EasyMock.mock(ConnectRestExtensionContext.class);
-        EasyMock.expect(context.configurable()).andReturn((Configurable) configurable);
-
-        EasyMock.replay(configurable, context);
-  
+        ArgumentCaptor<JaasBasicAuthFilter> jaasFilter = ArgumentCaptor.forClass(JaasBasicAuthFilter.class);
+        Configurable<? extends Configurable<?>> configurable = mock(Configurable.class);
+        when(configurable.register(any())).thenReturn(null);
```

Comment:
Why did we not use the captor here?

Reply:
Thank you for your reminders, moving captor from`Mockito.verify` to here is simpler.